runtime/ops/signal.rs

```diff
@@ -129,16 +129,16 @@ fn signal_str_to_int(s: &str) -> Option<libc::c_int> {
     "SIGUSR1" => Some(30),
     "SIGUSR2" => Some(31),
     _ => None,
-  }
+  })
 }
 
 #[cfg(target_os = "windows")]
-fn signal_str_to_int(_s: &str) -> Option<libc::c_int> {
-  unimplemented!()
+fn signal_str_to_int(_s: &str) -> Result<Option<libc::c_int>, AnyError> {
+  Err(generic_error("not implemented"))
 }
 
 pub fn signal_str_to_int_unwrap(s: &str) -> Result<libc::c_int, AnyError> {
-  signal_str_to_int(s)
+  signal_str_to_int(s)?
```

Comment:
After this change `signal_str_to_int_unwrap()` isn't really an appropriate function name any more.
Also, this is more complicated that it needs to be -- why not return `Result<c_int, AnyError>` from the various `signal_str_to_int()` implementations.

Reply:
Good point! I've updated the PR with all your feedback.